prime-router/src/main/kotlin/Element.kt

```diff
@@ -413,12 +418,37 @@ data class Element(
                     LocalDate.parse(formattedValue)
                     return null
                 } catch (e: DateTimeParseException) {
+                    // continue to the next try
                 }
                 try {
                     val formatter = DateTimeFormatter.ofPattern(format ?: datePattern, Locale.ENGLISH)
                     LocalDate.parse(formattedValue, formatter)
                     return null
                 } catch (e: DateTimeParseException) {
+                    // continue to the next try
+                }
+                try {
+                    // check for yyyyMMdd
+                    val formatter = DateTimeFormatter.ofPattern(datePattern, Locale.ENGLISH)
+                    LocalDate.parse(formattedValue, formatter)
+                    return null
+                } catch (e: DateTimeParseException) {
+                    // continue to the next try
+                }
+
+                // Cleanup the Date in variable values
+                val cleanedDate = formattedValue.replace("-", "/")
+                // the next five date validation patterns are valid date patterns that we have seen be
+                // manually entered into EMR systems, but are not consistent, so we cannot use the "format" param
+                val dateFormats = arrayOf("M/d/yyyy", "MMddyyyy", "yyyy/M/d", "M/d/yyyy HH:mm", "yyyy/M/d HH:mm")
```

Comment:
Move this to the companion object with the other date formats.  Then it will be a bit confusing why some date formats have their own variable and this set is an array.